Resources/Prototypes/Corvax/Entities/Structures/Wallmounts/Signs/posters.yml

```
@@ -0,0 +1,8 @@
+- type: entity
+  parent: PosterBase
+  id: PosterContrabandTrueEnemy
+  name: Истинный Враг
+  description: "Плакат намекающий на истинного врага Фронтира."
+  components:
+  - type: Sprite
+    sprite: /Corvax/Structures/Wallmounts/posters.rsi
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)